client/src/components/ProductInformation.jsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import StarReview from './StarReview.jsx';
 
 var ProductInformation = function(props) {
+  let { productInfo } = props;
+  let { default_price, category, name } = productInfo;
```

Comment:
I like the use of this destructuring pattern. It makes the data flow very clean.
I'll have to make use of this in other components.